src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -307,7 +321,12 @@ Here's what's happening behind the scenes:
 
 * Session replay works by rebuilding the user's DOM, which includes your website's structure and visual elements. To achieve this, our agent needs to access your CSS files to accurately capture styling information. We then inline these styles directly into the replay recording, eliminating the need to reference the original CSS files during playback.
 
-* If your CSS files are hosted on a separate domain, CORS may prevent our agent from accessing them. This can result in incomplete or inaccurate replays, as essential styling information may be missing.
+* If your CSS files are hosted on a separate domain, CORS may prevent our agent from accessing them. The agent attempts to fix the broken CSS. If it cannot, it sends the replay with broken styles along with attributes `hasCrossOriginCSS:true` and `inlinedCrossOriginCSS:false` so the UI knows some styles are broken and we can notify the user watching it with his to fix it. The agent is set up to report:
```

Comment:
@CherylFrankenfield @metal-messiah How exactly does the agent attempt to fix the broken css? How does it appear in the UI ("...so the UI knows some styles are broken and we can notify users of how to fix it...")?

Reply:
@CherylFrankenfield What does the UI link "See how to fix it" link to?